sale_product_set/models/product_set.py

```diff
@@ -8,4 +8,13 @@ class ProductSet(models.Model):
 
     name = fields.Char(_('Name'), help=_('Product set name'), required=True)
     set_line_ids = fields.One2many(
-        'product.set.line', 'product_set_id', _(u"Products"))
+        'product.set.line', 'product_set_id', _(u"Products"), copy=True)
```

Comment:
Why are you using `_` in field definition? Labels are already handled. This way, you'll get a static translation. Please remove it